components/quick_switch_modal/quick_switch_modal.jsx

```diff
@@ -311,7 +309,10 @@ export default class QuickSwitchModal extends React.PureComponent {
                         {help}
                     </div>
                     <SuggestionBox
-                        ref='switchbox'
+                        ref={(input) => {
```

Comment:
I'm fine with this but perhaps you want to create a function like

```js
suggestionBoxRef = (ref) => {
    this.switchBox = ref;
    this.focusTextbox();
}
```

and then here just do `ref={this.suggestionBoxRef}` just to avoid creating a new function each time this component renders

Reply:
Sure, but all the React docs show creating a new function :P